clients/common/cblas_interface.cpp

```diff
@@ -109,6 +110,32 @@ extern "C" {
 
     //axpy
     template<>
+    void cblas_axpy<rocblas_half>( rocblas_int n,
+                            rocblas_half alpha,
+                            rocblas_half *x, rocblas_int incx,
+                            rocblas_half *y, rocblas_int incy)
+    {
+        rocblas_int abs_incx = incx >=0 ? incx : -incx;
+        rocblas_int abs_incy = incy >=0 ? incy : -incy;
+        float alpha_float = static_cast<float>(alpha);
+        std::unique_ptr<float []> x_float(new float[n*abs_incx]());
```

Comment:
Using a std::unique_ptr<> for an array is fine, but I bet it would be easier to declare and you get array notation if you use std::array<>